doc/antora.yml

```diff
@@ -0,0 +1,7 @@
+name: isy-angular-widgets-doc
+title: isy-angular-widgets Dokumentation
+version: '17.2.x'
+display_version: '17.2.0'
```

Comment:
In der restlichen Doku werden nur Minor-Releaes angegeben. Hier würde also `17.2` als Version genügen.